test/common/README.md

```diff
@@ -761,6 +762,11 @@ a full `setImmediate()` invocation passes.
 should not be in scope when `listener.ongc()` is created.
 
 
+## tick Module
+
+The `tick` module provides a helper function that can be used to call a callback
+after a given number of event loop "ticks".
+
```

Comment:
should the documentation also (explicitly) mention the exported function name (`tick`) with input arguments ? I don't know what is the doc convention on modules with a single method

Reply:
Yes, I guess it would be nice to list the arguments here (and/or just provide an example of how this is used?)